client/state/data-layer/wpcom/account-recovery/lookup/index.js

```diff
@@ -32,12 +35,15 @@ export const validate = ( { primary_email, primary_sms, secondary_email, seconda
 	}
 };
 
-export const handleRequestResetOptions = ( { dispatch }, { userData } ) => (
+export const handleRequestResetOptions = ( { dispatch }, { userData }, next, transit = page ) => (
```

Comment:
Nice way to isolate this!

Should the redirect always happen after requesting the reset options? I feel like this redirect should happen as a result of some `submitUsername` UI action or similar - `handleRequestResetOptions` doesn't seem to have any relation to the UI but is triggering a route change.

Also, could we use a name like `redirect` instead of `transit`?